spec/features/create_new_work_spec.rb

```diff
@@ -17,14 +18,6 @@
     it 'deposits and renders work show page' do
       visit dashboard_path
 
-      click_link '+ Create a new collection'
-
-      fill_in 'Collection name', with: collection_attrs.fetch(:name)
-      fill_in 'Description', with: collection_attrs.fetch(:description)
-      fill_in 'Contact email', with: collection_attrs.fetch(:contact_email)
-
-      click_button 'Deposit'
-
       click_button '+ Deposit to this collection' # , match: :first
```

Comment:
And we know this is depositing into the collection on line 8 because it's the only collection that exists at this point, I guess?

Reply:
Correct.